api/src/test/java/org/openmrs/module/patientqueueing/api/PatientQueueingServiceTest.java

```diff
@@ -252,4 +252,30 @@ public void getMostRecentQueue_shouldReturnMostRecentPatientQueue() throws Parse
 		Assert.assertEquals(patientQueue, patientQueueingService.getMostRecentQueue(patient));
 		
 	}
+	
+	@Test
+	public void generateVisitNumber_shouldGenerateVisitNumberIrrespectiveOfPreviousVisitNumberLength() {
+		PatientQueueingService patientQueueingService = Context.getService(PatientQueueingService.class);
+		
+		Patient patient = Context.getPatientService().getPatient(10000);
+		
+		Location location = Context.getLocationService().getLocation(1);
+		
+		int standardVisitNumberLength = 18;
+		
+		PatientQueue patientQueue = new PatientQueue();
+		patientQueue.setPatient(patient);
+		patientQueue.setStatus(PatientQueue.Status.PENDING);
+		patientQueue.setEncounter(Context.getEncounterService().getEncounter(10000));
+		patientQueue.setLocationFrom(location);
+		patientQueue.setLocationTo(location);
+		patientQueue.setVisitNumber("20/10/2019-002");
+		patientQueueingService.savePatientQue(patientQueue);
+		
+		String visitNumber = patientQueueingService.generateVisitNumber(location, patient);
+		
+		Assert.assertNotEquals(patientQueue.getVisitNumber().length(), visitNumber.length());
```

Comment:
Why should the visit numbers not be equal?

Reply:
@dkayiwa that is comparing the length of the first test which validates the second test

Reply:
I just do not get it. To me the assertion looks as it is stating that no two visit numbers should be equal in length.

Reply:
Did you see the above comment?